xradar/io/backends/odim.py

```diff
@@ -477,6 +486,17 @@ def _sweep_number(self):
     def _odim_version(self):
         return self._get_odim_version()
 
+    @property
+    def _nyquist_velocity(self):
+        try:
+            nyquist_vel = self.how.get("NI", None)
+            if nyquist_vel is not None:
+                return float(nyquist_vel)
+        except (AttributeError, KeyError, TypeError):
+            pass
+
+        return None
```

Comment:
We can simplify this a bit, if we already catch the possible errors. If there are issues that "NI" might contain something which breaks the cast to float, we would need to add `ValueError` to that list.

```suggestion
        try:
            return float(self.how["NI"])
        except (AttributeError, KeyError, TypeError):
            return None
```